### DISCUSSION TITLE
When approximately will be a result of hackathon?

### DISCUSSION THREAD

**Discussion by YuriiMoroziukGunvor** [microsoft/hack-together-dotnet#114] [Q&A] [answered]
When approximately will be a result of hackathon?

**Comment by SimehZee** [comment]
Results already out [here](https://devblogs.microsoft.com/dotnet/great-dotnet-8-hack-winners/). Best overall is NASA Techport where Matt Soucoup is one of the contributors.

**Reply by codemillmatt** [reply, marked as answer]
Hi @SimehZee - you're correct that the NASA Techport app was the overall winner. However, I was not a contributor to the project https://github.com/tagr/greatnet8hack-techport/graphs/contributors

**Comment by YuriiMoroziukGunvor** [comment]
How to reciave digital badge ?

**Reply by SimehZee** [reply]
I had raised the same question, but have not received any response from the Hackathon folks.

**Reply by codemillmatt** [reply]
Badges should be coming soon!
